starknet_devnet/util.py

```diff
@@ -263,3 +264,62 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 # FeederGatewayClient is implemented in such a way that it logs and raises;
 # this suppresses the logging
 suppress_feeder_gateway_client_logger = LogSuppressor("services.external_api.client")
+
+logger = logging.getLogger("gunicorn.error")
+
+
+class LogContext(dict):
+    """Context manager for logging transactions accumulating info while executing transaction"""
+
+    def get_context_name(self) -> str:
+        """Getting name of current context to put as first line in log entry"""
+        return getattr(self, "name", None) or "Transaction"
+
+    def set_context_name(self, name):
+        """Setting name of current context to put as first line in log entry"""
+        setattr(self, "name", name)
+        return self
+
+    def cust_fees_to_int(self):
+        """Cust to human readable type"""
+        for key in ["actual_fee", "max_fee"]:
+            if self.get(key):
+                try:
```

Comment:
Looks hacky. If not rewritten, it could at least be better documented.

Reply:
removed this class